app/value_objects/has_nino_type.rb

```diff
@@ -0,0 +1,7 @@
+class HasNinoType < ValueObject
```

Comment:
I don't think `HasNino` should be converted into a type. It should only have the values 'yes', 'no', and 'nil'. Adding 'arc' as a value for `has_nino` makes sense only from the perspective of the form, not our underlying data model. Would it be possible to add both `has_nino` and `has_arc` if we need to store these values for rehydration? I realise it's a bit more work to update both values from the form, but I'm concerned that combining them into a single value just because they appear on the same form ties the data model too closely to a design that might change.